website/content/docs/connect/config-entries/mesh.mdx

```diff
@@ -193,6 +189,87 @@ spec:
 
 Note that the Kubernetes example does not include a `partition` field. Configuration entries are applied on Kubernetes using [custom resource definitions (CRD)](/docs/k8s/crds), which can only be scoped to their own partition.
 
+###  Peer Through Mesh Gateways
+
+Set the `PeerThroughMeshGateways` parameter to `true` to route peering control plane traffic through mesh gateways.
+
+<Tabs>
+<Tab heading="Consul OSS">
+
+<CodeTabs tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+
+```hcl
+Kind = "mesh"
+Peering {
+  PeerThroughMeshGateways = true
+}
+```
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: Mesh
+metadata:
+  name: mesh
+spec:
+  peering:
+    peerThroughMeshGateways: true
+```
+
+```json
+{
+  "Kind": "mesh",
+  "Peering": {
+    "PeerThroughMeshGateways": true
+  }
+}
+```
+
+</CodeTabs>
+
+</Tab>
+<Tab heading="Consul Enterprise">
+
+You can only set the `PeerThroughMeshGateways` attribute on `mesh` configuration entries in the `default` partition.
+This is the partition that owns mesh gateways routing control plane traffic to Consul servers.
```

Comment:
```suggestion
The `default` partition owns the traffic routed through the mesh gateway control plane to Consul servers.
```
Is that correct? I was having trouble parsing this sentence.

Reply:
You got it! 🙇